apps/site/src/app/layout.tsx

```diff
@@ -2,6 +2,7 @@ import type { Metadata } from "next";
 import water from "@/assets/backgrounds/water.jpg";
 import Footer from "@/lib/components/Footer/Footer";
 import "./globals.css";
+import Apply from "./apply/page";
```

Comment:
Nitpick: we have a certain order of importing things. See [this reply](https://github.com/HackAtUCI/irvinehacks-site-2024/pull/60#discussion_r1417792469) from Ian's PR.